src/components/OutboundWaDialog/OutboundWaDialog.js

```diff
@@ -31,51 +29,51 @@ import {
 } from '@twilio-paste/core';
 import Handlebars from 'handlebars';
 
-const AutoCompleteTemplate = (props) => {
-  const [value, setValue] = React.useState('');
+const AutoCompleteCombobox = (props) => {
+  const [value, setValue] = useState('');
+  const [displayItems, setDisplayItems] = useState(
+    props.inputItems.slice(0, props.maxDisplayedItems)
+  );
 
-  const clear = () => {
-    props.setSelectedItem('');
-    setValue('');
-    reset();
-  };
+  const handleInputValueChange = (inputValue) => {
+    if (inputValue !== undefined) {
+      const valueLowerCase = unidecode(value.toLowerCase());
 
-  const { reset, ...state } = useCombobox({
-    items: props.inputItems,
-    onSelectedItemChange: props.onSelectedItemChange,
-    onInputValueChange: ({ inputValue }) => {
-      if (inputValue !== undefined) {
-        props.setInputItems(inputValue);
-        setValue(inputValue);
-      }
+      const filteredItems = props.inputItems.filter((item) => {
+        const templateNameLowerCase = unidecode(
+          item.templateName.toLowerCase()
+        );
+        const messageLowerCase = unidecode(item.message.toLowerCase());
 
-      if (inputValue === '') {
-        clear();
-      }
-    },
-    inputValue: value,
-    selectedItem: props.selectedItem,
-  });
+        return (
+          templateNameLowerCase.includes(value) ||
+          messageLowerCase.includes(value)
+        );
+      });
+      setDisplayItems(filteredItems.slice(0, props.maxDisplayedItems));
+      setValue(inputValue);
+    }
+  };
+
+  const handleClearSelection = () => {
+    setValue('');
+    props.clear();
+    props.setSelectedItem('');
+    setDisplayItems(props.inputItems.slice(0, props.maxDisplayedItems));
+  };
 
   return (
     <>
       <Combobox
-        state={{ ...state, reset }}
-        items={props.inputItems}
-        required
-        disabled={props.loading}
```

Comment:
Parece que o disabled foi removido de forma inadvertida. Isso mantinha o checkbox desabilitado enquanto carregava

Reply:
Sim, caso você vá apagando letra por letra até o input ficar vazio, aparece todos os templates disponíveis.

Reply:
A gente não sabe o porquê do useCombobox provocar o bug de múltiplas renderizações, apenas constatamos que funciona sem ele, e não funciona com ele. Também vimos que na Twilio paste tem um warning sobre o uso do useCombobox, dizendo que é fácil de quebrar o sistema.
 
 _" 🚨Power user move!
Only use this property if you are a power user. It's very easy to break your implementation and unfortunately the Paste team will not be able to debug this for you. Proceed with extreme caution."_